e2e/cli-e2e/tests/collect.e2e.test.ts

```diff
@@ -78,8 +78,7 @@ describe('CLI collect', () => {
     expect(stderr).toBe('');
 
     expect(stdout).toContain('Code PushUp Report');
-    expect(stdout).toContain('Generated reports');
-    expect(stdout).toContain('report.json');
+    expect(stdout).not.toContain('Generated reports');
```

Comment:
This test scenario runs with `--persist.format=stdout`, that should be removed (no longer a valid option). And then I believe the assertions should stay the same, because JSON should be generated by default.